tests/unit/test_citations.py

```diff
@@ -255,10 +255,7 @@ def test_solver_citations(self):
             pybamm.IDAKLUSolver()
             self.assertIn("Hindmarsh2005", citations._papers_to_cite)
 
-    @unittest.skipIf(
-        system() == "Windows" or (system() == "Darwin" and "ARM64" in version()),
-        "JAX not supported on windows or Mac M1",
-    )
+    @unittest.skipIf(not(importlib.util.find_spec("jax")), "requires jax")
```

Comment:
create a `pybamm.have_jax()` function, for example in the jax solver, for this (see https://github.com/pybamm-team/PyBaMM/blob/989811b5633437ff69dd226d11053b2e36862a50/pybamm/solvers/idaklu_solver.py#L20)

Reply:
Added the have_jax in util because it was causing circular imports